aws/ec2.go

```diff
@@ -201,7 +201,7 @@ func getSubnets(svc ec2iface.EC2API, vpcID, clusterID string) ([]*subnetDetails,
 	// Fall back to full list of subnets if none matching expected tagging are found, with a stern warning
 	// https://github.com/kubernetes/kubernetes/blob/v1.10.3/pkg/cloudprovider/providers/aws/aws.go#L3009
 	if len(retFiltered) == 0 {
-		log.Printf("No tagged subnets found; considering all subnets. This is likely to be an error in future versions.")
+		log.Errorf("No tagged subnets found; considering all subnets. This is likely to be an error in future versions.")
```

Comment:
I would print this as `Info` for now as it is technically not defined as an error yet.